script.js

```diff
@@ -0,0 +1,8 @@
+const menuBar= document.querySelector(".menu-bar");
+const mobileMenu= document.querySelector(".mobile-menu");
+
+menuBar.addEventListener("click", () => {
+    menuBar.classList.toggle("active");
+    mobileMenu.classList.toggle("active");
+    closeIcon.classList.toggle("active");
+});
```

Comment:
- [ ] Kinldy **as suggested in the project requirements,** When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed, currently this function is not working 😨, 
 
   You may Fix this issue and make sure that also according to the project requirements When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears. currently, the mobile menu only disappears when the user clicks on the close button 😎😎😎
   
   https://www.loom.com/share/a407b481de814dce963743e045eab012